Accessibility Handbook.xcodeproj/project.pbxproj

```diff
@@ -1175,7 +1179,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 6;
 				DEVELOPMENT_ASSET_PATHS = "\"Accessibility Handbook/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 725HB8W2V6;
```

Comment:
I suggest not committing your development team

Reply:
my mistake 😅forgot it there